solcore/optics/external_optics.py

```diff
@@ -27,7 +73,8 @@ def solve_external_optics(solar_cell, options):
     except AttributeError as err:
         raise err
 
-    # We calculate the total amount of light absorbed in the solar cell, integrating over its whole thickness with a step of 1 nm
+    # We calculate the total amount of light absorbed in the solar cell, 
```

Comment:
Actually, this comment is no longer true and can be deleted as now the spacing is configurable in the input (I think the default is 1 angstrom)